### PR TITLE
Add back error when password is too short

### LINKED ISSUE #16802: [Bug]: Onboarding V2 Password length validation is not displayed
### Describe the bug

**Problem**: when I introduce a password which is not long enough, I don't see any warning, even though I cannot proceed.

https://user-images.githubusercontent.com/54408225/205687592-eca539de-90a3-481c-86cf-705a246d6f62.mp4

I am expecting an input validation message as we have in Onboarding V1

image

### Steps to reproduce

1. Load MM for the first time
2. Go through the onboarding process for Create a New Wallet
3. Introduce a short password 
4. Introduce the same password for confirmation
5. Click the consent checkbox

### Error messages or log output

_No response_

### Version

10.24.0

### Build type

None

### Browser

Chrome

### Operating system

Linux

### Hardware wallet

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by adonesky1**
In Onboarding V2 we do not show an error indicating why the user cannot proceed if they have entered a password shorter than 8 characters - we do show this error in Onboarding V1:
205687681-5f0a65fc-7ef3-4596-bc0b-028547d88413

* Fixes #16802

### Before

https://user-images.githubusercontent.com/34557516/206565268-705eee14-283d-4f75-aa2e-dabee309209d.mp4

### After

https://user-images.githubusercontent.com/34557516/206566750-d8c65a51-5c5a-4e03-8793-a360672ed0e6.mov

## Manual Testing Steps

1. Load MM for the first time
2. Go through the onboarding process for Create a New Wallet
3. Enter a password shorter than 8 characters
4. You should see an error - "Password is not long enough" - indicating why you cannot proceed.

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes

### CODE REVIEW

**Review by pedronfigueiredo — APPROVED** [review]

File: ui/pages/onboarding-flow/create-password/create-password.js [hunk 1]
```diff
@@ -100,11 +101,20 @@ export default function CreatePassword({
       </span>,
     ]);
 
+    if (passwordInput.length < 8) {
```
Comment (legobeat) [inline ui/pages/onboarding-flow/create-password/create-password.js:104]: Can this magic number be broken out into a configuration field or at least named constant like `PASSWORD_MIN_LENGTH`?

File: ui/pages/onboarding-flow/create-password/create-password.js [hunk 2]
```diff
@@ -85,13 +85,14 @@ export default function CreatePassword({
...
     let confirmError = '';
+    let passwordInputError = '';
     const passwordEvaluation = zxcvbn(passwordInput);
     const passwordStrengthLabel = getPasswordStrengthLabel(
       passwordEvaluation.score,
       t,
     );
-    const passwordStrengthDescription = passwordStrengthLabel.description;
-    const passwordStrengthInput = t('passwordStrength', [
+    let passwordStrengthDescription = passwordStrengthLabel.description;
```
Comment (legobeat) [inline ui/pages/onboarding-flow/create-password/create-password.js:94]: These could still be `const`s with some ternaries, like
`const passwordStrengtDescription = CONDITION ? X : Y`.

As it is right now the logic and intention is not trivial to follow.

**Comment by legobeat** [comment]
@adonesky1 broke out my suggestions into a PR targeting this one: #16902

**Review by jpuri — APPROVED** [review]
👍

**Pull request merged**